STLYEGUIDE_SPRESS_UPDATE/src/content/index.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: styleguide-default
+title: Styleguide
+---
+
+{% block content %}
+
+<div class="container sg-intro">
+  <div class="l-styleguide-column">
+
+    <header class='sg-header'>
+      <h1 class="sg-logo">
+        {{ site.title }}
+      </h1>
+    </header>
+
+    <main>
+      <h2>Living Styleguide</h2>
+      <p>This website acts as a living reference for the styles that exist on the live website for {{ site.title }}. The two sites share some design elements and components and deviate when necessary.</p>
+
```

Comment:
@aCyborg I find the second sentence in this description to be confusing: "The two sites. . . " It negates the idea that the client's website is built off of the style guide. Maybe just nix it?

Reply:
oh this is probably leftover from mount sinai, they had two sites.